build/dirs.proj

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="dir.props" />
+
+  <ItemGroup>
+    <Project Include="LocalizationTasks\Microsoft.Build.LocalizationTasks.csproj"
+             Condition="'$(FullFrameworkBuild)' == 'true'" />
```

Comment:
I added this condition, but is it the right one? Or is the right thing to build this project if we could use it (host == full framework), and always build it for the current host arch, overriding the build flavor?